runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -199,17 +218,35 @@ public static void addCacheGauges(@NonNull MetricRegistry metrics,
         }
     }
 
-    public static void addJvmMetrics(@NonNull MetricRegistry metrics, String pfx) {
+    public static void setupJvmMetrics(@NonNull MetricRegistry metrics) {
+        if (!metricsJvmCollectionEnabled) return;
+
         try {
-            metrics.register(pfx + "jvm.gc", metricsJVMGC);
-            metrics.register(pfx + "jvm.memory", metricsJVMMem);
-            metrics.register(pfx + "jvm.thread", metricsJVMThread);
-            metrics.register(pfx + "jvm.file-descriptors-used", metricsJVMFdGauge);
+            metrics.register("jvm.gc", metricsJVMGC);
+            metrics.register("jvm.memory", metricsJVMMem);
+            metrics.register("jvm.thread", metricsJVMThread);
+            metrics.register("jvm.file-descriptors-used", metricsJVMFdGauge);
         } catch (IllegalArgumentException e) {
             // Re-registering metrics during test runs, not a problem
         }
     }
 
+    private static void setupLogAnalysisMetrics(@NonNull MetricRegistry metrics) {
```

Comment:
Comments to tell me what this is for.

Reply:
Done. Also added more detailed comments and usage examples on loadProperties.